<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="windowTitle">
   <string>查找</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>查找内容：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editText"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkCase">
     <property name="text">
      <string>区分大小写</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnFindPrev">
       <property name="text">
        <string>向上查找</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFindNext">
       <property name="text">
        <string>向下查找</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>accept()</slot>
  </connection>
 </connections>
</ui>